src/core/index.ts

```diff
@@ -1,8 +1,6 @@
 import { performFetch, cyrb53, uuidv4 } from './utils';
 import { del, get, set } from './storage';
 
-import { Channel } from './channels';
```

Comment:
in this file few references to 'channel', e.g. ChannelSubAction. not used anymore?

Reply:
Probably not used but their usage is not trivial to track to safely remove. Will require some more effort, will do in another PR